### ISSUE TITLE
Rebuilding slack.com  Several People Are Coding

### ISSUE THREAD

**Issue by studiomohawk** [frontendweekly/feeds#2212]
[Rebuilding slack.com &ndash; Several People Are Coding](http://ift.tt/2gbNUpS) / Rebuilding slack.com A redesign powered by CSS Grid and optimized for performance and accessibility. Illustrations by Alice Lee . In August, we released a major&hellip;
